### PR TITLE
Bring over minimap icon from legacy oncoprint

### PR DESCRIPTION
**Pull request by adamabeshouse**
image

### CODE REVIEW

File: src/shared/components/oncoprint/controls/OncoprintControls.tsx [hunk 1]
```diff
@@ -838,8 +838,9 @@ export default class OncoprintControls extends React.Component<IOncoprintControl
                 <Button
                     active={this.showMinimap}
                     onClick={this.toggleShowMinimap}
+                    className="oncoprint__controls__minimap_button"
                 >
-                    <i className="fa fa-expand"></i>&thinsp;
+                    <img src="images/fitalteredcases.svg" alt="icon" width="20" height="20"/>&thinsp;
```
Comment (alisman) [inline src/shared/components/oncoprint/controls/OncoprintControls.tsx]: lets copy over this image and then use request() so that webpack will handle it

Comment (alisman) [reply src/shared/components/oncoprint/controls/OncoprintControls.tsx]: webpack will also probably automatically inline the svg into css file.

Comment (adamabeshouse) [reply src/shared/components/oncoprint/controls/OncoprintControls.tsx]: when you get back let's talk about how to do this

Comment (alisman) [reply src/shared/components/oncoprint/controls/OncoprintControls.tsx]: src/shared/components/mutationTable/column/FunctionalImpactColumnFormatter.tsx
line 49

**Pull request merged**